Kajabi/sage-lib
# [Sage-943] Type: New SageType component PoC

## Description

`SageType` Rails component is designed to allow developers to quickly output desired typography and UXD to control its styles with the Design System.

## Screenshot

![Screen Shot 2021-11-29 at 4 12 46 PM](https://user-images.githubusercontent.com/24237393/143951035-2931857b-aa97-43d1-b535-2edd9da9f015.png)

## Testing in `sage-lib`
1. Navigate to http://localhost:4000/pages/component/type and view the different options
2. Open `docs/app/views/examples/components/type/_preview.html.erb` and create some components of your own to test
3. Observe output for any issues or concerns.

## Testing in `kajabi-products`
1. (**LOW**) New component. No impact on existing `kajabi-products`.

## Related
Closes #943
